move_group/include/moveit/move_group/capability_names.h

```diff
@@ -46,6 +46,7 @@ static const std::string PLANNER_SERVICE_NAME = "plan_kinematic_path";    // nam
 static const std::string EXECUTE_SERVICE_NAME = "execute_kinematic_path"; // name of the advertised service (within the ~ namespace)
 static const std::string QUERY_PLANNERS_SERVICE_NAME = "query_planner_interface"; // name of the advertised query planners service
 static const std::string MOVE_ACTION = "move_group"; // name of 'move' action
+static const std::string EXECUTE_ACTION = "execute_action"; // name of 'execute' action
```

Comment:
i think "execute_path" or "execute_traj" would be better name. The other actions are not post-fixed with action